docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -0,0 +1,322 @@
+
+   - Set the **Mime type** to `image/jpeg` to configure image data capture.
+
+     {{< imgproc src="/tutorials/data-management/camera-data-capture.png" alt="The camera component configuration pane with data capture configuration enabled using type ReadImage and a capture frequency of 0.333" resize="600x" >}}
+
+1. Click **Save Config** at the bottom of the window to save your changes.
+
+For more information see [Configure data capture](/services/data/configure-data-capture/#configure-data-capture-for-individual-components) and [Configure cloud sync](/services/data/configure-cloud-sync/).
+
+### View and filter captured data
+
+Now that you have configured data capture on your camera component, you can view the resulting data files in the Viam app, and can apply filters to limit the files returned by specific criteria, such as by robot or date range.
+
+1. First, in the [Viam app](https://app.viam.com), browse to your robot's **Control** tab and enable the camera to verify that a live stream is returned.
+   If you do not see a functioning camera stream, verify that you have configured your [camera](/components/camera/) correctly.
+
+1. Then, select the [**DATA** page](https://app.viam.com/data/view) from the top of the screen.
+
+   {{< imgproc src="/tutorials/data-management/data-captured-images.png" alt="The data page showing a variety of captured images with search options presented on the left" resize="1200x" >}}
+
+   Here you can see the images captured so far from the camera on your smart machine.
+   New images should appear roughly every six seconds as cloud sync uploads them from your smart machine.
+
+You can use the filters on the left side of the screen to filter your images by robot, component, date range, and more.
+You can also use the **Copy export command** button to use the Viam CLI to [export captured data](/manage/cli/#data), if desired.
+
+For more information see [View and filter data](/manage/data/view/).
+
+## The ML model service
+
+Once your smart machine is capturing and syncing images to the Viam app, you are ready to train a new machine learning (ML) model using those image.
+You can use an ML model to help your smart recognize and adapt its behavior to the world around it.
+
+For this tutorial, you will train an ML model to be able to recognize specific objects, and then deploy that model to your smart machine using the _ML (machine learning) model service_.
+With a model deployed to your smart machine, the [ML model](/services/ml/) service can be used together with the [vision](/services/vision/) service to analyze newly-detected objects for a possible match to a known object.
+
+To train a new model from your captured data, first tag your images with appropriate labels, train a new model based on those labels, then deploy the model to your smart machine.
```

Comment:
bit much new
```suggestion
To train a model from your captured data, first tag your images with appropriate labels, train a model based on those labels, then deploy the model to your smart machine.
```

Reply:
You're right, no need to emphasize 'new' here, but also elsewhere. Removed 'new' in a bunch of places.